pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -463,6 +461,15 @@ func liveMigrationMonitor(vmi *v1.VirtualMachineInstance, dom cli.VirDomain, l *
 	acceptableCompletionTime := completionTimeoutPerGiB * getVMIMigrationDataSize(vmi)
 monitorLoop:
 	for {
+
+		select {
+		case passedErr := <-migrationErr:
```

Comment:
this is going to get triggered when migrationErr chan is closed, which may report as an error depending on the timing.

basically, ignore if passedErr == nil.

run this example and you'll see what I mean. 

```
package main

import (
	"fmt"
	"time"
)

func main() {

	c := make(chan error)

	go func() {
		for {
			select {
			case err := <-c:
				fmt.Printf("hit: %v\n", err)
				time.Sleep(time.Second)
			default:
				time.Sleep(time.Second)
			}
		}
	}()

	close(c)

	time.Sleep(5 * time.Second)

}
```

Reply:
Ah, got it. Thanks!